README.rst

```diff
@@ -95,3 +95,9 @@ Credits / Special Thanks
 - https://github.com/aaska
 
 Found Synology API "documentation" on this repo : https://github.com/kwent/syno/tree/master/definitions
+
+Official references:
+https://global.download.synology.com/download/Document/Software/DeveloperGuide/Package/Calendar/2.4/enu/Synology_Calendar_API_Guide_enu.pdf
+https://global.download.synology.com/download/Document/Software/DeveloperGuide/Package/FileStation/All/enu/Synology_File_Station_API_Guide.pdf
+https://global.download.synology.com/download/Document/Software/DeveloperGuide/Package/SurveillanceStation/All/enu/Surveillance_Station_Web_API.pdf
+https://global.download.synology.com/download/Document/Software/DeveloperGuide/Package/DownloadStation/All/enu/Synology_Download_Station_Web_API.pdf

```

Comment:
Make links a list + name list items like this \`[Service] API documentation\<link\>\`_

Order it alphabetically

Reply:
This is not resolved !

Reply:
Sorry, should be fixed now.